website/content/specification/syntax/definitions.md

```diff
@@ -61,7 +61,7 @@ The following example illustrates deprecating the flag named `flag-name` startin
 
 The `@name` attribute provides the definition's identifier, which can be used in other parts of a module, or in an importing *Metaschema module*, to reference the definition.
 
-This attribute's value MUST be a [token](/specification/datatypes/#token) that is unique among sibling top-level flag definitions.
+In top-level definitions, this attribute's value MUST be a [token](/specification/datatypes/#token) that is unique among sibling definitions of the same type.
```

Comment:
At first, I didn't understand why the word "flag" was in the sentence because this section is about names of flags, fields, and assemblies. My suggestion here is based on a conversation with @wendellpiez . If I'm changing the meaning by mistake, please let me know what the meaning is supposed to be.